src/libraries/Microsoft.PowerFx.Interpreter/RecalcEngine.cs

```diff
@@ -87,7 +96,7 @@ public static IExpression CreateEvaluatorDirect(CheckResult result)
             result.ThrowOnErrors();            
 
             (var irnode, var ruleScopeSymbol) = IRTranslator.Translate(result._binding);
-            return new ParsedExpression(irnode, ruleScopeSymbol);
+            return new ParsedExpression(irnode, ruleScopeSymbol, new StackMarker(20));

```

Comment:
>20 [](http://example.com/codeflow?start=81&length=2)

nit - should share this value.

Reply:
Value is now shared in an internal readonly static int in PowerFxConfig#DefaultMaxCallDepth.